src/MSBuild/Resources/Strings.resx

```diff
@@ -1225,6 +1225,24 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       LOCALIZATION: {0} contains the exception message explaining why the
       logger could not be created -- this message comes from the CLR/FX and is localized.
     </comment>
+  </data>
+  <data name="Process" Visibility="Public">
+    <value>Process = "{0}"</value>
+  </data>
+  <data name="MSBExePath" Visibility="Public">
+    <value>MSBuild executable path = "{0}"</value>
+  </data>
+  <data name="CommandLine" Visibility="Public">
+    <value>Command line arguments = "{0}"</value>
```

Comment:
I suggest not to quote, as with the arguments themselves quoted, it will get hard to read.

Reply:
Actually, I suggest that for all of these. We do not include quotes when dumping properties:

```
Build started 3/12/2021 3:41:46 PM.
Environment at start of build:
ALLUSERSPROFILE = C:\ProgramData
APPDATA = C:\Users\danmose\AppData\Roaming
CommonProgramFiles = C:\Program Files\Common Files
CommonProgramFiles(x86) = C:\Program Files (x86)\Common Files
CommonProgramW6432 = C:\Program Files\Common Files
...
```
we should be consistent. It will be clear whether there are quotes in the value.